gapis/replay/manager.go

```
@@ -153,13 +162,32 @@ func (m *manager) createScheduler(ctx context.Context, device bind.Device) {
 	m.mutex.Lock()
 	defer m.mutex.Unlock()
 	m.schedulers[deviceID] = scheduler.New(ctx, deviceID, m.batch)
+	ch := make(chan executeArgs, 16)
+	m.executors[deviceID] = ch
+	crash.Go(
+		func () {
+			for {
+				args := <-ch
+				// TODO(pmuetschard): Make the "state reconstruction" an actual event status and have gapir let us
+				// know how far along it is in executing it. Also, there is technically still some server work
+				// happening after this (e.g. storing the payload in the db), but that is fast and passing things
+				// down isn't worth it.
+				// This transitions the status from building to executing.
+				args.replayStatus.Progress(ctx, 0, 1, 0)
+				ctx = status.Start(ctx, "Execute Replay")
+				args.exec.execute(ctx, m, args.key)
```

The error code here is swallowed and not returned to anyone.